gcloud/_helpers.py

```diff
@@ -394,6 +394,27 @@ def _to_bytes(value, encoding='ascii'):
         raise TypeError('%r could not be converted to bytes' % (value,))
 
 
+def _from_bytes(value):
```

Comment:
So there is a companion function to this called `_to_bytes`.  Maybe this should be `_to_unicode`. I'll change it to that unless you think of a better name.